cmd/containerExecuteStructureTests.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+type containerExecuteStructureTestsUtils interface {
+	Stdout(out io.Writer)
+	Stderr(err io.Writer)
+	RunExecutable(e string, p ...string) error
+	Glob(pattern string) (matches []string, err error)
+}
+
+type containerExecuteStructureTestsUtilsBundle struct {
+	*command.Command
+	*piperutils.Files
+}
+
+func newContainerExecuteStructureTestsUtils() containerExecuteStructureTestsUtils {
+	utils := containerExecuteStructureTestsUtilsBundle{
+		Command: &command.Command{},
+		Files:   &piperutils.Files{},
+	}
+	utils.Stdout(log.Writer())
+	utils.Stderr(log.Writer())
+	return &utils
+}
+
+func containerExecuteStructureTests(config containerExecuteStructureTestsOptions, _ *telemetry.CustomData) {
+	utils := newContainerExecuteStructureTestsUtils()
+	err := runContainerExecuteStructureTests(&config, utils)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func findConfigFiles(pattern string, utils containerExecuteStructureTestsUtils) ([]string, error) {
+	files, err := utils.Glob(pattern)
+	if err != nil {
+		return nil, err
+	}
+
+	return files, nil
+}
+
+func runContainerExecuteStructureTests(config *containerExecuteStructureTestsOptions, utils containerExecuteStructureTestsUtils) error {
+	utils.Stdout(log.Writer())
+	utils.Stderr(log.Writer())
+	containerStructureTestsExecutable := "container-structure-test"
+	var parameters []string
+	parameters = append(parameters, "test")
+	configFiles, err := findConfigFiles(config.TestConfiguration, utils)
```

Comment:
`testConfiguration` should be either flagged mandatory or with a proper default value